src/parsley/parsley.py

```
@@ -38,26 +41,66 @@
+
+    Where decoding succeeds, string names are converted to Enum members from
+    `parsley.message_types`. Unknown values remain as hexified strings.
+    """
+    # use the lower-level parse which returns strings/hex fallbacks
+    parsed = parse(msg_sid, msg_data)
+
+    # attempt to convert to Enum members where possible
+    def to_enum(enum_cls, name: str):
+        try:
+            return enum_cls[name]
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)